cafebabel/static/js/edit-article.js

```diff
@@ -0,0 +1,6 @@
+const tagButtonAdd = document.querySelector('.tags button.add')
+const tagButtonRemove = document.querySelector('.tags button.remove')
+tagButtonAdd.addEventListener('click', (event) => {
```

Comment:
I see this is stopping the native behavior, but it seems like it's doing nothing instead.
Ane what is the default behavior of a button anyways?